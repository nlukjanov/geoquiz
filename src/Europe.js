import React from 'react';

const Europe = () => {

  return <div data-testid='europe'>Hello from Europe</div>;
};

export default Europe;
