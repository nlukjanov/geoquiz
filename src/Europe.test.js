import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Europe from './Europe';

describe('<Europe />', () => {
  it('should render a component', () => {
    const { container, getByTestId } = render(
      <BrowserRouter>
        <Europe />
      </BrowserRouter>
    );
    expect(getByTestId('europe')).toBeTruthy;
    expect(container.firstChild).toMatchSnapshot();
  });

});
