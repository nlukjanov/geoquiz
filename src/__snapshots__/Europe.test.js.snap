// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Europe /> should render a component 1`] = `
<div
  data-testid="europe"
>
  Hello from Europe
</div>
`;
